### PR TITLE
Monokle documentation update

### PR DESCRIPTION
**Pull request by iamrajeev**
This PR...

## Changes

-

## Fixes

-

## How to test it

-

## screenshots

-

## Checklist

- [ ] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

File: docs/features.md [hunk 1]
```diff
@@ -39,3 +39,23 @@ resource, Monokle will quickly show you, that you have a dead link with a warnin
 
 Want to browse your cluster instead of a repo? Simply smash the "Show Cluster Objects" button to import all objects from
 the cluster into Monokle - [read more](cluster-integration.md)
+
+
+## Reload Folder On Startup
```
Comment (olensmar) [inline docs/features.md:44]: none of these three need to be in the features files - lets move them as follows:
- move "Reload Folder On Startup" and "Navigation History" to the resource-navigation.md file
- move "Editing kustomization.." to both kustomize.md and helm.md

File: docs/helm.md [hunk 2]
```diff
@@ -49,3 +49,8 @@ Monokle is now in "Preview Mode" (as indicated by the header at the top):
 - selecting "Preview" for a different values file will preview the output of Helm for that file instead
 - selecting "Exit" in the top right or next to the values file restores the previous resource navigator.
 
+## Editing kustomizations and Helm Values
```
Comment (olensmar) [inline docs/helm.md:52]: this section should only talk about Helm - not kustomize

File: docs/kustomize.md [hunk 3]
```diff
@@ -66,3 +66,8 @@ specified in the kustomization configuration:
 
 ![Kustomize Preview Content](img/kustomize-preview-content.png)
 
+## Editing kustomizations and Helm Values
```
Comment (olensmar) [inline docs/kustomize.md:69]: this section should only talk about Kustomize, not Helm

File: docs/helm.md (hunk 2 above)
Comment (iamrajeev) [reply docs/helm.md]: Resolved

File: docs/kustomize.md (hunk 3 above)
Comment (iamrajeev) [reply docs/kustomize.md]: Resolved.

**Review by olensmar — APPROVED** [review]

**Pull request merged**